src/NexusMods.App.UI/Pages/MyGames/MyGamesViewModel.cs

```diff
@@ -89,6 +94,12 @@ public MyGamesViewModel(
         this.WhenActivated(d =>
             {
                 gameRegistry.InstalledGames
+                    .Where(game =>
+                    {
+                        if (experimentalSettings.EnableAllGames) return true;
+                        return experimentalSettings.SupportedGames.Contains(game.Game.GameId);
+                    })
+                    .ToReadOnlyObservableCollection()
```

Comment:
We could check if game is already managed, and only hide if it isn't managed and it isn't supported

Reply:
Too overkill in my opinion.
My games view no longer adds loadouts; like it used to back in the day.

The loadouts view is still unrestricted.
So functionally, there wouldn't be much difference here.

Reply:
Wait... do we not have a way to remove individual loadouts anymore?
I don't see that anywhere in the UI.

Reply:
Oh, I see. You can only delete loadouts that aren't the first loadout. So you need to still be able to see managed games in `My Games`, to unmanage them. Seems AL's right here; we do need to show these.